
---
uint32 vehicle_id